www/src/content/docs/components/Otel-middleware.mdx

```diff
@@ -0,0 +1,81 @@
+---
+title: Otel middleware
+description: description of otel middleware
+---
+
+The Hono [Otel middleware](https://github.com/fiberplane/fpx/tree/main/packages/client-library-otel) leverages [OpenTelemetry](https://opentelemetry.io/) to instrument applications and send traces to FPX Studio. It requires the use of [Hono](https://hono.dev/) web application framework.
+
+## Installation 
+To install the Otel middleware for your Hono application, run the following command in your project directory:
+```
+npm i @fiberplane/hono-otel@beta 
+``` 
+
+## Usage within the Hono application
+Below is an example of how to use the Otel middleware within a Hono application:
+```typescript
+import { Hono } from "hono";
+import { instrument } from "@fiberplane/hono-otel";
+
+const app = new Hono();
+
+app.get("/", (c) => c.text("Hello, Hono!"));
+
+export default instrument(app);
+```
+
+### Advanced usage: Custom spans
+The measure function from the Hono Otel middleware allows you to create custom spans for any function within your application, enabling more granular tracing.
+
+```typescript
+import { instrument, measure } from "@fiberplane/hono-otel";
+
+const app = new Hono();
+
+// Create a loop function that will get recorded as a span inside the trace for a incoming given request
+const loop = measure("loop", (n: number) => {
+  for (let i = 0; i < n; i++) {
+    console.log(`Loop iteration: ${i}`);
+  }
+});
+
+app.get("/", (c) => {
+  loop(100);
+  return c.text("Hello, Hono!");
+});
+
+export default instrument(app);
+```
+
+## Configuration
+By default, the Otel Hono middleware does not send traces. To define an endpoint, you need to set the `FPX_ENDPOINT` environment variable.
+
+When starting Fiberplane Studio via the command line with `npx @fiberplane/studio`, the `FPX_ENDPOINT` is automatically set to the Fiberplane Studio's default port 8788: `http://localhost:8788/v1/traces`. Additionally, the Fiberplane Studio CLI setup enables Node.js compatibility mode for Cloudflare Workers.
+
+If you are not using Fiberplane Studio, ensure that Node.js compatibility mode is enabled in the `wrangler.toml` file:
```

Comment:
Worth noting: `wrangler.toml` is specific to Cloudflare setups

Reply:
yes true, I will add a line saying it is only needed for cloudflare workers